src/pybind/mgr/dashboard/frontend/src/app/shared/components/bottom-links/bottom-links.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'cd-bottom-links',
+  templateUrl: './bottom-links.component.html',
+  styleUrls: ['./bottom-links.component.scss']
+})
+export class BottomLinksComponent {
```

Comment:
Since this is no longer used in the bottom, maybe we should change the name.
```suggestion
export class InfoLinksComponent {
```
or 
```suggestion
export class LegalLinksComponent {
```

Reply:
Will make the change.

Reply:
Done.